frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -292,23 +292,19 @@ export const StyledNoLogoSpacer = styled.div(({ theme }) => ({
 
 export interface StyledSidebarOpenContainerProps {
   chevronDownshift: number
-  isCollapsed: boolean
 }
 
 export const StyledSidebarOpenContainer =
   styled.div<StyledSidebarOpenContainerProps>(
-    ({ theme, chevronDownshift, isCollapsed }) => ({
+    ({ theme, chevronDownshift }) => ({
       position: "fixed",
       top: chevronDownshift ? `${chevronDownshift}px` : theme.spacing.xl,
-      left: isCollapsed ? theme.spacing.twoXL : `-${theme.spacing.twoXL}`,
+      left: theme.spacing.twoXL,
```

Comment:
Removing the conditional rendering also eliminated the need for the container to move out of the frame when collapsed (using negative position -> `-${theme.spacing.twoXL}`)